CauseFactory.sol

```diff
@@ -33,7 +33,8 @@ contract CauseFactory {
         return infos;
     }
 
-    function cfRetrieveIds() public view returns (string[] memory) {
```

Comment:
Could you leave this function in? I've taken a look at the docs and apprently you can only access one value of an array or mapping at a time, so returning the entire array is more efficient...

Reply:
Okay🫡

Reply:
Could you uncomment the code pls and then I'll approve the commit

Reply:
Sorry! I thought I had uploaded the ucommented file 2 days ago, but it seems like the file didn't successfully uploaded at that time. 
It should works now!